### DISCUSSION TITLE
How to clear/reset a FeatureState

### DISCUSSION THREAD

**Discussion by vimception** [mrpmorris/Fluxor#471] [Q&A]
Is there an idiomatic way to easily clear/reset/re-initialise a particular FeatureState? With that I mean re-creating the state object and informing any listener.

Of course, this can be done by creating an (empty) action and a corresponding reducer which just creates a new state. But this seems like a use case/pattern which is common enough I would expect library support for it.

Some cases when this would be useful:
- When the user signs out from an application, you want to make sure all states are reset.
- When something changes in state A and this invalidates state B.

Thanks in advance for any pointers.

**Comment by mrpmorris** [comment]
Have a public static readonly field on the class named Empty.

Then dispatch an action that has a reducer to set the state back to empty.
